web/satellite/src/components/accessGrants/AccessGrants.vue

```diff
@@ -24,6 +24,97 @@
                 />
             </div>
         </div>
+        <div v-if="isNewAccessGrantFlow" class="access-grants__new-title-area">
```

Comment:
For `access-grants__title-area` (the old one), could you add `v-if="!isNewAccessGrantFlow"` so that we don't have two title areas if the new flow is enabled?

Reply:
addressed in previous commit